node/core/runtime-api/src/lib.rs

```diff
@@ -447,6 +452,8 @@ where
 		),
 		Request::CandidateEvents(sender) =>
 			query!(CandidateEvents, candidate_events(), ver = 1, sender),
+		Request::SessionEeParamsByParentHash(sender) =>
+			query!(SessionEeParamsByParentHash, session_ee_params_by_parent_hash(), ver = 2, sender), // FIXME: Bump API vesrion?
```

Comment:
@tdimitrov should I put `ver = 3` here as well?

Reply:
Yes, but please don't hardcode it. Add your message [here](https://github.com/paritytech/polkadot/blob/11221b5a5636e1896b63700767a5f2891643bbe7/node/subsystem-types/src/messages.rs#L709) and use the constant on as much places as you can. Effectively this means everywhere but `#[api_version(X)]` attribute macro.

You can search for `DISPUTES_RUNTIME_REQUIREMENT` to see an example.